applications/GeoMechanicsApplication/tests/test_partially_saturated/test_saturated_below_phreatic_level_pw_triangle3N/ProjectParameters_stage1.json

```diff
@@ -0,0 +1,191 @@
+      "solver_type"                    : "LinearSolversApplication.sparse_lu",
+      "scaling"                        : true
+    },
+    "problem_domain_sub_model_part_list": ["Parts_Solid_Grond"],
+    "processes_sub_model_part_list"     : ["gravity", "WATER_PRESSURE_Initial", "WATER_PRESSURE_Bottom", "WATER_PRESSURE_Top"],
+    "body_domain_sub_model_part_list"   : ["Parts_Solid_Grond"],
+    "newmark_beta"                     : 0.25,
+    "newmark_gamma"                    : 0.5,
+    "newmark_theta"                    : 0.5,
+    "rayleigh_m"                       : 0.0,
+    "rayleigh_k"                       : 0.0,
+    "move_mesh_flag"                   : false
+  },
+  "output_processes": {
+    "gid_output": [
+      {
+        "python_module"                : "gid_output_process",
+        "kratos_module"                : "KratosMultiphysics",
+        "process_name"                 : "GiDOutputProcess",
+        "Parameters": {
+          "model_part_name"            : "PorousDomain.porous_computational_model_part",
+          "output_name"                : "stage1",
+          "postprocess_parameters": {
+            "result_file_configuration": {
+              "gidpost_flags": {
+                "WriteDeformedMeshFlag": "WriteUndeformed",
+                "WriteConditionsFlag"  : "WriteElementsOnly",
+                "GiDPostMode"          : "GiD_PostAscii",
+                "MultiFileFlag"        : "SingleFile"
+              },
+              "file_label"             : "step",
+              "output_control_type"    : "step",
+              "output_interval"        : 1,
+              "body_output"            : true,
+              "node_output"            : false,
+              "skin_output"            : false,
+              "plane_output"           : [],
+              "nodal_results"          : ["DISPLACEMENT", "TOTAL_DISPLACEMENT", "WATER_PRESSURE", "VOLUME_ACCELERATION",
+			                              "LINE_LOAD_Y", "TOTAL_STRESS_TENSOR", "CAUCHY_STRESS_TENSOR"],
+              "gauss_point_results"    : ["GREEN_LAGRANGE_STRAIN_TENSOR", "ENGINEERING_STRAIN_TENSOR", "VON_MISES_STRESS",
+                                          "FLUID_FLUX_VECTOR", "HYDRAULIC_HEAD", "DEGREE_OF_SATURATION"]
+            },
+            "point_data_configuration" : []
+          }
+        }
+      }
+    ]
+  },
+  "processes": {
+    "constraints_process_list": [
+      {
+        "python_module"                : "apply_scalar_constraint_table_process",
+        "kratos_module"                : "KratosMultiphysics.GeoMechanicsApplication",
+        "process_name"                 : "ApplyScalarConstraintTableProcess",
+        "Parameters": {
+          "model_part_name"            : "PorousDomain.WATER_PRESSURE_Initial",
+          "variable_name"              : "WATER_PRESSURE",
+          "is_fixed"                   : false,
+          "table"                      : 0,
+          "fluid_pressure_type"        : "Phreatic_Multi_Line",
```

Comment:
This type aims at phreatic lines that consist of multiple line segments. This type works, but there is a simpler one that is targeted at your situation, where the phreatic line has a uniform $`y`$ coordinate.

Reply:
I think one of the goals of of these test cases is to prove that multi-line phreatic line works correctly. 
What do you think @WPK4FEM ?

Reply:
The objective of the tests is to show that the correct equilibrium is found and stays stable. The method used for providing the pressure boundary condition can be any, preferably the simplest.